### ISSUE TITLE
create relationship between this and Operate First initiative

### ISSUE THREAD

**Issue by goern** [redhat-edge-computing/blueprint-management-hub#12]
looking thru the repo it appears that this is very similar with what the Op1st group is doing, how can we learn from each other? What could be reused?

@durandom @harshad16

**Comment by durandom** [comment]
Agreed. I'm not 100% sure what the purpose of this repo is and if it's implemented/deployed somewhere.
https://github.com/operate-first/ is trying to come up with repeatable best practices. Let's discuss how we could converge
